etc/phing-grammar.rng

```diff
@@ -3373,6 +3373,9 @@
                     <attribute name="haltOnError"/>
                     <attribute name="source"/>
                 </optional>
+                <optional>
+                    <attribute name="verbose"/>
```

Comment:
What do you think about describing the default value i.e. with `a:defaultValue="false"` maybe?